### PR TITLE
Robot End-Effector Cameras

### PR DESCRIPTION
**Pull request by jaybrecht**
Added the ability for users to add RGB or RGBD cameras to the robot end effectors through the user sensor configuration yaml file. 

robot_camera

* Created visual meshes for camera and mount 
* Added xacro arguments for the camera
* Updated vacuum gripper macro to include gazebo sensors based on xacro arguments
* modified ariac.launch.py to parse sensor config to pass camera info when creating the robot description
* modified test competitor sensor config to reflect necessary changes

### CODE REVIEW

File: ariac_sensors/src/ariac_camera_plugin.cpp
Comment (saahu27) [inline ariac_sensors/src/ariac_camera_plugin.cpp:1]: Is this the change to publish images once the competition is started?

**Review by saahu27 — COMMENTED** [review]
Just Checking?  if the change in the ariac sensors  camera plugin is to publish only once the comp is started?

File: ariac_sensors/src/ariac_camera_plugin.cpp
Comment (jaybrecht) [reply ariac_sensors/src/ariac_camera_plugin.cpp:1]: yes this was a bug before, all the sensors are supposed to wait before the competition starts before publishing data

Comment (saahu27) [reply ariac_sensors/src/ariac_camera_plugin.cpp:1]: Gotcha! Thanks Justin. I have checked in rviz and everything works as intended. Would you like to merge the changes to dev?

**Review by saahu27 — APPROVED** [review]

**Pull request merged**